e2e/settings.e2e-spec.ts

```diff
@@ -8,10 +8,10 @@ import { UploadPageObject } from './page-objects/upload-page-object';
 describe('Settings E2E Test', () => {
 
   let originalTimeout;
-  let settingsPage = new SettingsPageObject();
+  const settingsPage = new SettingsPageObject();
   let loginPage = new LoginPageObject();
```

Comment:
Thanks correct i "fixed" those in the review  and realized they have been initalizied twice